source/features/small-user-avatars.tsx

```diff
@@ -11,7 +11,7 @@ function addAvatar(link: HTMLElement): void {
 	const username = link.textContent;
 	const size = 14;
 
-	link.classList.add('d-inline-block');
+	link.classList.add('d-inline-block', 'rgh-lh-initial');
```

Comment:
GitHub already has some classes:

```css
.lh-condensed-ultra {
    line-height:1 !important
}

.lh-condensed {
    line-height:1.25 !important
}

.lh-default {
    line-height:1.5 !important
}

.lh-0 {
    line-height:0 !important
}
```

Reply:
Changed to `lh-condensed-ultra`,  effect is the same